src/components/date-picker/date-picker-icons.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+export const CalendarIcon: React.SFC<React.HTMLAttributes<SVGElement>> = props => {
+    return (
+        <svg
```

Comment:
if this SVG element should not get focus add attribute `focusable="false"` for IE

Reply:
most of these attributes can and should be moved to the css class